README.md

```diff
@@ -1135,6 +1135,28 @@ k = Kafka.new(
 )
 ```
 
+or use a STS client to retrieve temporary AssumeRoleCredentials. AssumeRoleCredentials will be updated in the background automatically.
```

Comment:
Can you make some of these terms links to the relevant AWS docs? First time users would benefit from that.